api/desecapi/pdns.py

```diff
@@ -90,11 +90,16 @@ def get_keys(domain):
     """
     Retrieves a dict representation of the DNSSEC key information
     """
+    def _filter_ds(key):
+        key['ds'] = [ds for ds in key['ds'] if int(ds.split()[2]) in [2, 4]]
```

Comment:
consider putting that in settings.py or at least in a local/module variable so that it becomes really obvious what is done here

Reply:
This piece of code will go away with pdns 4.5, and the behavior is then governed by PowerDNS' publish-cds setting (which we already use for CDS records). See https://github.com/PowerDNS/pdns/pull/10217

As this is not permanent, I prefer to keep as is